app/design/frontend/base/default/template/algoliasearch/instantsearch/hit.phtml

```diff
@@ -14,61 +14,61 @@ $price_key = $config->isCustomerGroupsEnabled(Mage::app()->getStore()->getStoreI
 
 <!-- Product hit template -->
 <script type="text/template" id="instant-hit-template">
-    {{#hits}}
```

Comment:
Huge BC break, I wouldn't modify this template. I'd rather create a new template like `infinite-scroll-hit.phtml` and copy it.
It'll be harder for the maintenance, but better for BC POV, what do you think?

Or, you can create a template `simple-hit.phtml` and include it here in `hit.phtml`, so we would have code written only once with preserving BC. Hmm?